systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -385,6 +397,157 @@ void testCruiseControlMetrics() {
         }
     }
 
+    @Test
+    void testKafkaMetricsSettings() {
```

Comment:
So what does this actually test? I'm not entirely sure after reading the test. Maybe it should start with some Javadoc explaining it.